source/opcode.cpp

```diff
@@ -236,10 +236,10 @@ int32_t spvOpcodeIsPointer(const SpvOp opcode) {
   switch (opcode) {
```

Comment:
-> other comment below
(I'd rather not add checks for everything that could possibly ever return a pointer in physical mode, so either always return true in physical mode here, or have "logical" in the function name)